<launch>
  <node name="joy_avoidance" pkg="rab_bringup" type="joy_avoidance" output="screen">
    <param name="sub_cmdvel_topic" value="/cmd_vel_old" />
    <param name="scan_topic" value="/base_scan" />
    <param name="pub_cmdvel_topic" value="/cmd_vel" />
  </node>
</launch>